### ISSUE TITLE
keyboard shortcut screen cannot be scrolled by keyboard

### ISSUE THREAD

**Issue by feonixrift** [nolanlawson/pinafore#1473]
I attempt to scroll up and down in the keyboard shortcut help screen (activated by '?') but nothing happens. j,k do not work; up and down arrows do not work; page up and down keys do not work. I had to use the mouse to learn the keyboard shortcuts. Is there a possibility to add keyboard scrolling to this help view? Thanks.
